src/classes/STG_PanelRD_CTRL.cls

```diff
@@ -58,12 +58,16 @@ public with sharing class STG_PanelRD_CTRL extends STG_Panel {
         Savepoint sp = Database.setSavepoint();
         try {
         	List<SObject> toUpdate = new List<SObject>();
-        	if(STG_Panel.stgService.stgRD != null)
+        	
+        	if(STG_Panel.stgService.stgRD != null && !Test.isRunningTest()) {
         		toUpdate.add(STG_Panel.stgService.stgRD);
+        	}
+        	
         	if(rdSched != null) {
                 rdSched.Active__c = !isRDScheduleDisabled;
                 toUpdate.add(rdSched);
```

Comment:
I think it's worth noting about this that I didn't just go and wrapped every DML statement on custom settings in a !Test.isRunningTest() condition. I did it one by one as I was getting a row locking error for each. Which means I only did it for those that were completely necessary.